hypertrace-alert-engine/src/main/java/org/hypertrace/alert/engine/RuleEvaluationJobManager.java

```diff
@@ -97,9 +97,7 @@ public void stopJob(Scheduler scheduler) throws SchedulerException {
   }
 
   private void addAlertTasksToJobData(JobDataMap jobDataMap, Config appConfig) {
-    RuleSource ruleSource =
-        RuleSourceProvider.getProvider(
-            appConfig.getConfig(AlertTaskJobConstants.JOB_DATA_MAP_RULE_SOURCE));
+    RuleSource ruleSource = RuleSourceProvider.getProvider(appConfig.getConfig("alertRuleSource"));
```

Comment:
Currently hard coded the name(here and at other places) but there could be a decent way.

Reply:
define it in AlertTaskJobConstants

Reply:
name ?